### DISCUSSION TITLE
Should probably remove Jupyter notebook examples or put them in a separate project

### DISCUSSION THREAD

**Discussion by aia** [OpenBB-finance/OpenBB#552] [General]
With a recent check-in (https://github.com/GamestonkTerminal/GamestonkTerminal/pull/531) of Jupyter notebook, Github is classifying the project as HTML because of the size of notebooks :-)

![image](https://user-images.githubusercontent.com/535247/125988668-74319e58-a7d4-4ae9-8aa5-95cc6db50881.png)

![image](https://user-images.githubusercontent.com/535247/125988819-9f131e45-fb35-4329-afdc-7dd197ab6f74.png)

I think we should remove them.

**Comment by DidierRLopes** [comment]
Check #554 solution. What do you think?
